shop/src/main/java/ro/msg/learning/shop/controller/OrderController.java

```diff
@@ -0,0 +1,24 @@
+package ro.msg.learning.shop.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.DTO.OrderDTO;
+import ro.msg.learning.shop.model.Order;
+import ro.msg.learning.shop.service.OrderService;
+
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    @PostMapping("/orders/{customerID}")
```

Comment:
The RequestMapping from line 12 is a prefix for all the method mappings. No need to add it again in the PostMapping because it will be duplicated.